Dockerfile

```diff
@@ -1,19 +1,32 @@
-FROM debian:stable
+FROM python:3.6-jessie
 
-MAINTAINER Clifton Barnes <clifton.a.barnes@gmail.com>
+MAINTAINER Clifton Barnes <clifton.barnes@rovercode.com>
 
 RUN apt-get update
-RUN apt-get install -y python python-dev python-pip python-smbus
+RUN apt-get install -y python3-dev libi2c-dev
 
-EXPOSE 80
+# https://www.linuxcircle.com/2015/05/03/how-to-install-smbus-i2c-module-for-python-3/
+# TODO: Remove this build once python3-smbus is not needed
+WORKDIR /tmp
+RUN wget http://ftp.de.debian.org/debian/pool/main/i/i2c-tools/i2c-tools_3.1.0.orig.tar.bz2
```

Comment:
I'm hoping we can find a better location to get this from; a repo would be ideal.